### ISSUE TITLE
Unexpected EOF for dynamically 

### ISSUE THREAD

**Issue by andreaswachowski** [apollographql/eslint-plugin-graphql#156]
Consider the following use case: On the UI, a list of several items with a checkbox next to each one. You activate a number of check boxes, then click a submit button to store the selected items. The submit shall trigger *one* call to the GraphQL server.

To this end, our application dynamically builds a mutation string of the form

```
mutation createMultipleItems {
  createItem1: createItem(item: {
    attr1: 'value1'
    attr2: 'value2'
    ...
  }
  createItem2: createItem(item: {
    attr21: 'value21'
    attr22: 'value22'
    ...
  }
  ...
  createItemN: createItem(...)
}
```

Thanks to GraphQL, the server knows how to disassemble this and executes each `createItem` mutation in turn. But this is just to illustrate that we need to dynamically build the query, since we don't know the number of items selected in the UI in advance.

Once the query is assembled, we have literally the following code line:

```js
const mutation = gql`${queryString}`;
```

This yields, unexpectedly
```
error  Syntax Error: Unexpected <EOF>                    graphql/template-strings
```

I presume in this case there's not much that eslint-plugin-graphql can evaluate, so I suppose it should just ignore such cases (or emit a warning that it cannot analyze this)?

**Comment by jnwng** [comment]
it'd be useful to see a broken example of this (e.g., what's the value of the `queryString` variable?)

in this case, you're right, we should probably just ignore this case since there's no viable graphql document. we might be able to check that before we ingest so we can just ignore this entirely...
